stdlib/zipfile.pyi

```diff
@@ -13,6 +14,16 @@ error = BadZipfile
 
 class LargeZipFile(Exception): ...
 
+class _ZipStream(Protocol):
+    def read(self, __n: int) -> bytes: ...
+    # The following methods are optional:
+    # def seekable(self) -> bool: ...
+    # def tell(self) -> int: ...
+    # def seek(self, __n: int) -> Any: ...
+
+class _ClosableZipStream(_ZipStream, Protocol):
+    def close(self) -> Any: ...
```

Comment:
```suggestion
    def close(self) -> object: ...
```
This should allow callers to return anything, but avoid an explicit `Any`.

Reply:
That's a good alternative, will change. I wish we had a `Void` type to make this explicit.